lib/Backend/Commands.hs

```diff
@@ -241,25 +241,45 @@ findCmd (FindOptions pathMb textMb sortMb filters filterFields) = do
       OpLTE -> (<=)
       OpEQ -> (==)
 
-renameCmd :: RenameOptions -> Sem r RenameResult
-renameCmd (RenameOptions _oldPath _newPath _force) = do
-  -- TODO: reuse copy+delete operations
-  undefined
+renameCmd
+  :: forall r.
+     ( Member BackendEffect r
+     , Member (Embed IO) r
+     , Member (Error CofferError) r
+     , Member (Error RenameResult) r
+     )
+  => RenameOptions -> Sem r RenameResult
+renameCmd (RenameOptions oldPath newPath force) = do
+  operations <- buildCopyOperations oldPath newPath force
+  void $ runCopyOperations operations
+  -- we don't want to delete clashed entries if renaming is forced.
+  let pathsToDelete =
+        flip filter operations \(CopyOperation old _) -> none (\(CopyOperation _ new) -> old ^. path == new ^. path) operations
```

Comment:
This line is way too long (see [the style guide here](https://github.com/serokell/coffer/blob/main/docs/code_style.md#line-length))

Reply:
Done